dist/src/editor/editor-mathfield.js

```diff
@@ -2521,12 +2521,19 @@ MathField.prototype.complete_ = function(options) {
     return false;
 }
 
+MathField.prototype._getSuggestions = function(command) {
+    const suggestions = Definitions.suggest(command);
+    if (!this.config.suggestionWhitelist) {
```

Comment:
Where is this whitelist defined? Is it defined inside projects that are using this library?

Reply:
@gavin-brown yeah, you would just add it to the `config` object that is passed to MathField constructor, e.g. `{suggestionWhitelist: ['\\sin']}`

Reply:
the `src/core/definitions` file contains _all_ the possible command definitions fyi